gsadjust/pyqt_models.py

```diff
@@ -1616,39 +1632,32 @@ class DatumTableModel(QtCore.QAbstractTableModel):
     Model to store Datums, shown on the adjust tab.
     """
 
+    _headers = { # As map, so do not need to be kept in order with the above.
+        DATUM_STATION: 'Station',
+        DATUM_G: 'g',
+        DATUM_SD: 'Std. dev.',
+        DATUM_DATE: 'Date',
+        DATUM_TIME: 'Time (UTC)',
+        MEAS_HEIGHT: 'Meas. height',
+        GRADIENT: 'Gradient', 
+        DATUM_RESIDUAL: 'Residual', 
+        N_SETS: '# sets',
+    }
+
+    signal_adjust_update_required = QtCore.pyqtSignal()
+
     def __init__(self):
         super(DatumTableModel, self).__init__()
         self.datums = []
 
-    signal_adjust_update_required = QtCore.pyqtSignal()
-
     def headerData(self, section, orientation, role=QtCore.Qt.DisplayRole):
         if role == QtCore.Qt.TextAlignmentRole:
             if orientation == QtCore.Qt.Horizontal:
                 return QVariant(int(QtCore.Qt.AlignLeft | QtCore.Qt.AlignVCenter))
             return QVariant(int(QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter))
-        if role != QtCore.Qt.DisplayRole:
-            return QVariant()
-        if orientation == QtCore.Qt.Horizontal:
-            if section == DATUM_STATION:
-                return QVariant("Station")
-            elif section == DATUM_G:
-                return QVariant("g")
-            elif section == DATUM_SD:
-                return QVariant("Std. dev.")
-            elif section == DATUM_DATE:
-                return QVariant("Date")
-            elif section == DATUM_TIME:
-                return QVariant("Time (UTC)")
-            elif section == MEAS_HEIGHT:
-                return QVariant("Meas. height")
-            elif section == GRADIENT:
-                return QVariant("Gradient")
-            elif section == DATUM_RESIDUAL:
-                return QVariant("Residual")
-            elif section == N_SETS:
-                return QVariant("# sets")
```

Comment:
You don't need to use `QVariant` when returning strings (old PyQt API if I remember correctly).